WordPress/Classes/ViewRelated/Blog/Subscribers/Helpers/SubscribersBlog.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import WordPressShared
+import WordPressKit
+
+@MainActor
```

Comment:
What does this `@MainActor` do? The functions and properties in this struct do not appear to be bound to the main thread.

Reply:
I synchronized everything on `@MainActor` just to make sure`. It does get initialized with a main-thread confined `Blog` entity.